programs/sol-xr/src/instructions/close_minting_round.rs

```diff
@@ -0,0 +1,56 @@
+use crate::mint_round::{AssociatedRoundAccount, MintRound};
+use crate::{Invest, InvestBumps};
+use anchor_spl::metadata::Metadata;
+use std::ops::Div;
+use {
+    crate::state::sol_strategy::SolStrategy,
+    anchor_lang::prelude::Rent,
+    anchor_lang::prelude::*,
+    anchor_lang::system_program,
+    anchor_spl::{
+        associated_token::AssociatedToken,
+        token::{mint_to, Mint, MintTo, Token, TokenAccount},
+    },
+};
+
+#[derive(Accounts)]
+pub struct CloseMintingRound<'info> {
+    #[account(
+        mut,
+        constraint = sol_strategy.governance_authority.key() == governance_authority.key() @ Error::UnauthorizedGovernanceAuthority,
+    )]
+    pub governance_authority: Signer<'info>,
+    #[account(
+        mut,
+        seeds = [SolStrategy::SEED_PREFIX],
+        bump
+    )]
+    pub sol_strategy: Account<'info, SolStrategy>,
+    #[account(
+        init_if_needed,
```

Comment:
im not sure why we need `init_if_needed` when closing a mint round. Shld be removed if not needed for safety